bookwyrm/templates/settings/reports/report.html

```diff
@@ -61,30 +62,59 @@ <h3 class="title is-4">{% trans "Reported links" %}</h3>
 {% include 'settings/users/user_moderation_actions.html' with user=report.user %}
 {% endif %}
 
-<div class="block">
-    <h3 class="title is-4">{% trans "Moderator Comments" %}</h3>
-    {% for comment in report.reportcomment_set.all %}
-    <div class="card block">
-        <p class="card-content">{{ comment.note }}</p>
-        <div class="card-footer">
-            <div class="card-footer-item">
-                <a href="{{ comment.user.local_path }}">{{ comment.user.display_name }}</a>
+<div class="block content">
+    <h3 class="title is-4">{% trans "Moderation Activity" %}</h3>
+
+    <div class="box">
+        <ul class="mt-0">
+            {% for comment in report.reportaction_set.all %}
+            <li class="mb-2">
+                <div class="is-flex">
+                    <p class="mb-0 is-flex-grow-1">
+                        {% if comment.action_type == "comment" %}
+                            {% blocktrans trimmed with user=comment.user|username user_link=comment.user.local_path %}
+                            <a href="{{ user_link }}">{{ user}}</a> commented on this report:
+                            {% endblocktrans %}
+                        {% else %}
+                            {% blocktrans trimmed with user=comment.user|username user_link=comment.user.local_path %}
+                            <a href="{{ user_link }}">{{ user}}</a> took an action on this report:
+                            {% endblocktrans %}
+                            <span class="has-text-weight-bold">
+                                {{ comment.get_action_type_display }}
+                            </span>
+                        {% endif %}
+                    </p>
+                    <span class="tag">{{ comment.created_date }}</span>
+                </div>
+
+                {% if comment.note %}
+                <blockquote>{{ comment.note }}</blockquote>
+                {% endif %}
+            </li>
+            {% endfor %}
+
+            <li class="mb-2">
```

Comment:
Suggest moving this to the top, otherwise everything runs top to bottom except the initial opening of the report, which is kinda weird.